### DISCUSSION TITLE
Website down?

### DISCUSSION THREAD

**Discussion by BBrown4** [JedWatson/react-select#4524] [General]
Hey y'all - I can't seem to get https://react-select.com to come up? Is the website down?

**Comment by BBrown4** [comment]
Welp, seems to have loaded now but it took a solid 5 minutes or so for it to actually navigate and load (it was not my local internet connection)

**Reply by ebonow** [reply]
It should be up and running now. I am not seeing the same slowness you are though it was def out earlier this morning.

**Reply by BBrown4** [reply]
Yep, working just fine now, thanks for getting back to me!

**Comment by ebonow** [comment]
Looks like here was the culprit
https://www.netlifystatus.com/
